src/OpenTelemetry.Exporter.Jaeger/Implementation/Process.cs

```diff
@@ -45,6 +47,50 @@ public Process(string serviceName, IDictionary<string, object> processTags)
 
         public List<JaegerTag> Tags { get; set; }
 
+        public Resource LibraryResource { get; private set; }
```

Comment:
I don't understand why you need either of these here. You can set the Process service name and other process tags directly from the udp batcher on first append right?  This class care nothing about Resource. All it cares about is a valid service name and a bunch of tags which could be anything.

You can probably revert your changes to this class, IJaegerUdpBatcher, JaegerTraceExporter and simply do this manipulation from the JaegerUdpBatcher itself.

Also, if somebody sets the ServiceName on the Process directly during registration we might want to honor that I think.

Reply:
@pcwiese Good call! I couldn't encapsulate the logic in `JaegerUdpBatcher` because it only sees `JaegerSpan` instances which are stripped of `LibraryResource` when converted from `SpanData`. I decided to move the `ApplyLibraryResource` method all the way up to `JaegerTraceExporter`. Much cleaner but I couldn't escape modifying the `IJaegerUdpBatcher` interface completely because `JaegerTraceExporter` needs access to at least the `Process` instance to do its work. LMK what you think about the current state. If we add `LibraryResource` to `JaegerSpan` then we could make it work exactly as you described.

> Also, if somebody sets the ServiceName on the Process directly during registration we might want to honor that I think.

I used the same logic as ZipkinExporter. It will take Resource over Options. I'm happy to change, but please confirm you still want it given that behavior.

Reply:
Actually... might be good to stick with `Process` exposed on `IJaegerUdpBatcher` so that if someone wants to build their own batcher and plug it in they will have their `Process` instance automatically populated from LibraryResource by the parent exporter? Otherwise they will have to re-implement similar logic.

Reply:
makes sense.